tests/test_jffs.py

```diff
@@ -0,0 +1,9 @@
+from common_helper_extraction.jffs import extract_jffs
+
+from .helper import get_binary_from_test_file
+
+
+def test_extract_fs():
```

Comment:
does this test test anything that is not tested in `test_fs_extraction()`?

Reply:
No, but i feel like keeping one functional test per module, to keep the test and its testee in a formal bound. The redundancy can be solved without damaging the code, but I think we would loose readability.